src/rust/src/datatypes.rs

```diff
@@ -145,15 +145,15 @@ impl PlRDataType {
         name.try_into().map_err(savvy::Error::from)
     }
 
-    pub fn new_decimal(scale: Option<NumericScalar>, precision: Option<NumericScalar>) -> Result<Self> {
+    pub fn new_decimal(
+        scale: Option<NumericScalar>,
+        precision: Option<NumericScalar>,
+    ) -> Result<Self> {
```

Comment:
Just modified by `cargo fmt`

Reply:
In general, do not commit changes of unrelated files.